migration/create-b-cases/select-temp-cto-table.sql

```diff
@@ -0,0 +1,8 @@
+SELECT ce.id AS collectionexerciseid, cto.actionplanid
```

Comment:
Rather than going the CSV route, you could `SELECT blahblah INTO casesvc.temp_cto` (and probably turn this into a one-script, non-manual migration in the process)

Reply:
Afraid we can't do this. In Prod/Preprod the tables are in different databases so this can't be done in one script.

Reply:
That makes me sad :( oh well!